helm/templates/core-deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: telemetry-core
+  labels:
+    {{- include "substrate-telemetry.labels" . | nindent 4 }}
+    telemetry-component: core
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount.core }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "substrate-telemetry.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "substrate-telemetry.selectorLabels" . | nindent 8 }}
+        telemetry-component: core
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "substrate-telemetry.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - "telemetry_core"
+          args:
+            - "--listen"
+            - "0.0.0.0:8000"
```

Comment:
Will there be custom args that could be passed through here ?

Reply:
There definitely are other args that could be used, but I'm not sure how much I care about being able to pass them in via values.yaml versus having to update the helm chart to change them personally :)

Reply:
Generally this helm chart is tailored for this application and it doesn't mean to be a general helm chart for deploying multiple substrate-telemetry deployments. So I think adding the args (if any) to this file manually would make sense atm.